tests/test_protocols.py

```diff
@@ -143,4 +146,29 @@ def test_extract_rabi():
         extract_rabi(RabiAmplitudeEFData)
 
 
+def test_resonator_flux_bias():
+    freq_width = 100_000
+    freq_step = 10_000
+    bias_width = 1
+    bias_step = 0.2
+    flux_start = -0.5
+    flux_end = 0.4
+    flux_step = 0.1
+    ResonatorFluxParameters(freq_width, freq_step, bias_width, bias_step)
+    ResonatorFluxParameters(freq_width, freq_step, flux_start, flux_end, flux_step)
+    try:
+        ResonatorFluxParameters(
+            freq_width,
+            freq_step,
+            bias_width,
+            bias_step,
+            flux_start,
+            flux_end,
+            flux_step,
+        )
+    except ValueError:
+        # Cannot initialize both bias and flux amplitude sweeps
+        pass
```

Comment:
```suggestion
    with pytest.raises(ValueError):
        ResonatorFluxParameters(
            freq_width,
            freq_step,
            bias_width,
            bias_step,
            flux_start,
            flux_end,
            flux_step,
        )
```
Just avoid the `try-except-pass` block :)